android/app/src/main/java/org/fossasia/openevent/data/Event.java

```diff
@@ -16,77 +16,84 @@
 
     @PrimaryKey
     private Integer id;
-    @JsonProperty("event_url")
     private String eventUrl;
     private String description;
-    @JsonProperty("licence_details")
     private LicenseDetails licenseDetails;
-    @JsonProperty("start_time")
     private String startTime;
-    @JsonProperty("end_time")
     private String endTime;
-    @JsonProperty("ticket_url")
     private String ticketUrl;
     private String topic;
     private String thumbnail;
     private String timezone;
     private String logo;
-    @JsonProperty("searchable_location_name")
     private String searchableLocationName;
     private String large;
-    @JsonProperty("background_image")
     private String backgroundImage;
-    @JsonProperty("call_for_papers")
     private CallForPapers callForPapers;
-    @JsonProperty("location_name")
     private String locationName;
     private String name;
     private Copyright copyright;
     private String privacy;
-    @JsonProperty("placeholder_url")
     private String placeholderUrl;
-    @JsonProperty("social_links")
     private RealmList<SocialLink> socialLinks;
     private Double longitude;
-    @JsonProperty("organizer_name")
     private String organizerName;
-    @JsonProperty("schedule_published_on")
     private String schedulePublishedOn;
     private String state;
     private Version version;
-    @JsonProperty("has_session_speakers")
     private Boolean hasSessionSpeakers;
-    @JsonProperty("sub_topic")
     private String subTopic;
     private Double latitude;
-    @JsonProperty("organizer_description")
     private String organizerDescription;
     private String identifier;
     private String type;
     private String email;
-    @JsonProperty("code_of_conduct")
     private String codeOfConduct;
 
+    @JsonGetter("event_url")
```

Comment:
No need to annotate getters, prefer JsonProperty there. Getter annotation creates unnecessary boilerplate. Only annotate setters when there are more than one attribute name for same property. JsonProperty should be preferred over JsonSet/Getter. But we have to delegate to setter when there are more than one attribute name, but there is no need of getter annotation as we don't serialize the POJOs, only deserialize